api/locations/data/locationType.enum.ts

```diff
@@ -0,0 +1,6 @@
+export enum LocationTypeEnum {
+  // Hearing location
```

Comment:
You could name the variable with a prefix off hearingLocation unnecessary comment

Reply:
I dissagreed this I would prefer not put in any necessary prefix to keep the enum key as short as possible.
For thie comment out code it's for reference because we have two location type one for Hearing and another for Work allocation.